chall01/aadlercr.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+
+import sys
+
+
+def printMorse(message):
+    morse = [
```

Comment:
I would recommend put that out of the function, to avoid redeclaring/assigning this variable everytime you call printMorse()